simulator/simple_planning_simulator/package.xml

```diff
@@ -21,6 +23,7 @@
   <depend>lanelet2_extension</depend>
   <depend>motion_utils</depend>
   <depend>nav_msgs</depend>
+  <depend>raw_vehicle_cmd_converter</depend>
```

Comment:
I feet it is too much to depend on the `raw_vehicle_cmd_converter` only for the `csv_loader`. Would you have the code in the vehicle model file?